tests/conftest.py

```diff
@@ -1282,3 +1282,9 @@ def duts_minigraph_facts(duthosts, tbinfo):
         }
     """
     return duthosts.get_extended_minigraph_facts(tbinfo)
+
+@pytest.fixture(scope="module", autouse=True)
+def get_reboot_cause(duthost):
+    yield
+    reboot_cause_history_got = duthost.show_and_parse("show reboot-cause history")
```

Comment:
When will the ```reboot_cause_history_got``` be checked?

Reply:
It will be checked at the end of each test case.

Reply:
I mean is it only logged into file but never checked?

Reply:
Yes, only logged.

Reply:
I see. Then I agree with Xin's idea. The call of ```duthosts.shell``` will generate DEBUG log.
Actually, the test gap described in #4430 doesn't make too much sense to me.  In my idea, a fixture like check system uptime will be more helpful for detecting unexpected reboot. I'll comment in that issue and get more input.

Reply:
ok